src/solvers/smt2/smt2_parser.cpp

```diff
@@ -1135,6 +1135,28 @@ void smt2_parsert::command(const std::string &c)
       return;
     }
 
+    auto &entry = id_map[id];
+    entry.type = sort();
+    entry.definition = nil_exprt();
+  }
+  else if(c == "declare-var") // this appears to be accepted by Z3 and CVC4
```

Comment:
Why not just do `if(c == "declare-const" || c == "declare-var")` above and then use `c` in the error message?